src/PerfView.TestUtilities/ThrowingTraceListener.cs

```diff
@@ -21,6 +21,7 @@ public sealed class ThrowingTraceListener : TraceListener
     {
         public override void Fail(string message, string detailMessage)
         {
+            Xunit.Assert.True(false, message + Environment.NewLine + detailMessage);
```

Comment:
💭 I would not expect this line to be necessary, but it also shouldn't hurt anything.

Reply:
I like this line because it insures that no one can swallow the exception, and the error 'stops' a bit closer to where it happened than if you allow the exception to propagate (and presumably cause the test system to ultimately fail.

Reply:
I'm not sure that's the case. `Assert.True` just throws [a different exception](https://github.com/xunit/assert.xunit/blob/dbb0b75cc8fa9349d63a97d2af453d2cd568faaf/BooleanAsserts.cs#L95) at the same location where we were already throwing one.

Reply:
That makes me sad, and seems like a dubious thing for a test harness to do.   It is pretty important that test harnesses don't ignore failures REGARDLESS  of what the code under test does (it may swallow all exceptions, the test harness should not care).   When a Xunit.Assert fails it should be impossible for the test to succeed, period.  It is pretty easy for it to do (it just remembers that it failed).     Something to chase down at some  point...